package.json

```diff
@@ -43,10 +43,9 @@
     "test": "jasmine --config=./tests/jasmine.json",
     "cc": "npm run build && nyc jasmine --config=./tests/jasmine.json",
     "lint": "tslint --fix --project .",
-    "start": "node dist/src/core.js",
-    "core": "node dist/src/core.js",
-    "bitcoin": "node dist/src/bitcoin.js",
-    "ipfs": "node dist/src/ipfs.js"
```

Comment:
For my understanding, any reason for the rename here?

Reply:
Before the start and core were duplicates of the same script, felt simpler to just have one script prefix instead of start followed by which service you want to start?